binding/HarfBuzzSharp.Tizen/HarfBuzzSharp.Tizen.csproj

```diff
@@ -18,7 +20,7 @@
     <DebugType>pdbonly</DebugType>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="MSBuild.Sdk.Extras" Version="1.6.65" PrivateAssets="All" />
```

Comment:
Was this intentional?

Reply:
Yes because Sdk.Extras was removed from the project at some point so this isn't needed or am I wrong?